app/api/speakers.py

```diff
@@ -81,7 +81,13 @@ def before_post(self, args, kwargs, data=None):
             and not data.get('email')
         ):
             data['email'] = current_user.email
-
+        if (
+            not has_access('is_organizer', event_id=data['event'])
```

Comment:
`has_access('is_organizer', event_id=data['event'])` is accessed already, so save it in a variable and reuse elsewhere